public/app/core/utils/fetch.ts

```diff
@@ -7,19 +7,20 @@ export const parseInitFromOptions = (options: BackendSrvRequest): RequestInit =>
   const isAppJson = isContentTypeApplicationJson(headers);
   const body = parseBody(options, isAppJson);
 
-  if (options?.withCredentials) {
```

Comment:
this is a big breaking change, withCredentials is a boolean in the our BackendSrv request api

Reply:
I think the name convention of `withCredentials` comes from [XMLHttpRequest.withCredentials](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials) property which has true or false value and `credentials` comes from fetch API [Request.credentials](https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials) property and has `omit`, `include` and `same-origin`. In order to keep the parameter(like the options variable) consistency, it is better to use credentials instead of withCredentials. Since modern browsers with latest version use `same-origin` as the default value, we may explicitly set it(make browser with old version works in the same way). 
The `options.credentials` used in many places in grafana frontend source code, so change the request api directly may not be a good idea. But I recommend use `same-origin` as the default value and use can override by passing [`return await this.request({ method: 'GET', credentials: 'include', url, params, requestId });`](https://github.com/grafana/grafana/blob/1e8b10ef057863b009299f70d0634a65090dcc75/public/app/core/services/backend_srv.ts#L358); @torkelo